modules/reitit-core/src/reitit/impl.cljc

```diff
@@ -85,6 +85,15 @@
             routes)
       (not-empty)))
 
+(defn unresolved-conflicts [path-conflicting]
+  (-> (into {}
+            (remove (fn [[[_ route-data] conflicts]]
+                      (and (:conflicting route-data)
+                           (every? (comp :conflicting second)
+                                   conflicts)))
+                    path-conflicting))
```

Comment:
[golf] You could use a transducer here for tiny performance gains. `(into {} (remove ...) path-conflicting)`.

Reply:
Done.